File: BaekJoon/Bronze/4153/C++/4153.cpp
// no.4153: 직각삼각형 (B3)

#include <cstdio>
using namespace std;

int main() {
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);
    while(!(a==0 && b==0 && c==0)) {
        if(a*a + b*b == c*c || a*a + c*c == b*b || b*b + c*c == a*a) {
            printf("right\n");
        } else {
            printf("wrong\n");
        }
        scanf("%d %d %d", &a, &b, &c);
    }
    return 0;
}

File: BaekJoon/Bronze/4153/Python/4153.py
import sys

a, b, c = map(int, sys.stdin.readline().split())
while not (a == 0 and b == 0 and c == 0):
    if a >= b and a >= c:
        if b ** 2 + c ** 2 == a ** 2:
            print('right')
        else:
            print('wrong')
    elif b >= a and b >= c:
        if a ** 2 + c ** 2 == b ** 2:
            print('right')
        else:
            print('wrong')
    else:
        if a ** 2 + b ** 2 == c ** 2:
            print('right')
        else:
            print('wrong')
    a, b, c = map(int, sys.stdin.readline().split())